lib/monetary.ts

```diff
@@ -207,4 +208,15 @@ export class ETHAmount extends MonetaryAmount<Ethereum> {
   }
 }
 
-export class ERC20Amount extends MonetaryAmount<ERC20> {}
+export class ERC20Amount extends BaseMonetaryAmount<ERC20> {}
+
+export interface ExchangeRate<From extends Currency, To extends Currency> {
```

Comment:
Is this the exchange rate normalized to 1 unit of the `From` currency?

Reply:
That's what I have in mind.